packages/strapi-plugin-content-manager/admin/src/components/CustomTable/Row.js

```diff
@@ -91,6 +91,14 @@ function Row({ canDelete, canUpdate, isBulkable, row, headers }) {
     {
       icon: canUpdate ? <FontAwesomeIcon icon="pencil-alt" /> : null,
     },
+    {
+      icon: canUpdate ? <FontAwesomeIcon icon="clone" /> : null,
+      onClick: e => {
+        e.stopPropagation();
+        emitEvent('willCloneEntryFromList');
```

Comment:
I am not sure we want this event. @sam-pires

Reply:
Is it for the telemetry?

Reply:
so, do I delete it ?

Reply:
Yes please.